### PR TITLE
SRGAN added

### PR DESCRIPTION
**Pull request by spazewalker**

### CODE REVIEW

**Comment by nirmal-suthar** [comment]
- Rename folder resources to assets
- You may delete pre-trained models, cause it takes much space. (if you have download link for weights, then you can simply make user download it for use)
- Also, move all files from srgan folder to root folder (It is much cleaner removing srgan folder)

**Comment by spazewalker** [comment]
proposed changes were commited

File: generative_models/SRGAN_tensorflow/model.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+from tensorflow.python.keras.layers import Add, BatchNormalization, Conv2D, Dense, Flatten, Input, LeakyReLU, PReLU, Lambda
+from tensorflow.python.keras.models import Model
+from tensorflow.python.keras.applications.vgg19 import VGG19
+from utils import subpixel_conv2d,normalize_01,normalize_m11,normalize,denormalize,denormalize_m11
+
+LR_SIZE = 24
+HR_SIZE = 96
```
Comment (nirmal-suthar) [inline generative_models/SRGAN_tensorflow/model.py]: add this arg to parser

**Review by nirmal-suthar — APPROVED** [review]

File: generative_models/SRGAN_tensorflow/model.py (hunk 1 above)
Comment (spazewalker) [reply generative_models/SRGAN_tensorflow/model.py]: okay

**Comment by nirmal-suthar** [comment]
Can you move this to new folder named super_resolution

File: generative_models/SRGAN_tensorflow/README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# Tensorflow Implementation of SRGAN
+
+## Usage
+```bash
+$ python3 main.py -h
+```
+following are the arguments:
+```bash
```
Comment (nirmal-suthar) [inline generative_models/SRGAN_tensorflow/README.md:8]:
```suggestion
```

```

**Review by nirmal-suthar — APPROVED** [review]

**Comment by spazewalker** [comment]
> Can you move this to new folder named super_resolution

okay

**Pull request merged**